src/tool_writeout_json.c

```diff
@@ -149,7 +156,7 @@ static int writeVersion(FILE *str, CURL *curl, const char *key, CURLINFO ci)
 }
 
 void ourWriteOutJSON(const struct writeoutvar mappings[], CURL *curl,
-        struct OutStruct *outs, FILE *stream)
+        struct per_transfer *per, FILE *stream)
```

Comment:
While changing this line, you can indent it the way we do: the s in the first `struct` should be aligned with the c from `const` in the line above!